components/DesktopFilterView.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
```

Comment:
Can you show me an example for what you mean? What do you mean by uniform? I adjusted the media query width on here so it will just turn them into buttons to get rid of this issue

Reply:
@vsauceda-cfa this is what I mean by the filter become larger. Maybe you make them each a bit bigger and add an `overflow: hidden;` rule?

![Screen Shot 2021-07-30 at 2 50 37 PM](https://user-images.githubusercontent.com/72828520/127715281-fce4b5f9-1b64-4a0a-8cc5-d53c79ea67d2.png)

and this is what I mean by the elements running together when the view gets smaller.

![Screen Shot 2021-07-30 at 2 50 53 PM](https://user-images.githubusercontent.com/72828520/127715322-e04175ef-6754-41c8-b70b-46662011239a.png)